dashboard/UI-Test/test.yaml

```diff
@@ -0,0 +1,65 @@
+sequence:
+- testcase 1
+- testcase 2
+test:
+- testcase 1:
+  - scenario: invalid pbench-dashboard login
+  - step 1:
+      action: goto
+      target: http://10.1.170.201
```

Comment:
This structure seems a little...odd.  It makes reasonable sense that a `test` would be a sequence of "testcases". It's slightly weird that each testcase would be a mapping with a single key mapped to a sequence, although I suppose that lets you store the name of the test case as its key instead of an attribute.  But, it's really weird that the first entry in the testcase sequence is a "scenario" while the other entries are "steps".

A more natural structure might look like this:
```
test:
- name: testcase 1
  scenario: invalid pbench-dashboard login
  steps:
  - name: step 1
    action: goto
    target: http://10.1.170.201
    ...
  ...
...
```
That is, `test` is a sequence of test cases, each of which is a mapping containing a name, scenario, and a sequence of steps; each step is a mapping containing a step name, an action, a target, and so forth.

Although, if you really like the "name attribute as key", it might look like this:
```
test:
- testcase 1:
    scenario: invalid pbench-dashboard login
    steps:
    - step 1:
        action: goto
        target: http://10.1.170.201
        ...
    ...
...
```
where `test` is a sequence of test cases, each of which is a mapping of a single key to a mapping containing a scenario and a sequence of steps; each step is a mapping of a single key to a mapping containing an action, a target, and so forth.